lib/taiko.js

```diff
@@ -229,6 +229,7 @@ const checkIfElementIsCovered = async (elem, x, y, elems, selector) => {
         isElemAtPoint = await checkIfChildOfOtherMatches(elems, parseInt(x), parseInt(y));
     if (!isElemAtPoint)
         throw Error(description(selector) + ' is coverred by other element');
```

Comment:
Guess this will break throwing error for the first element itself and not continue, has to be done after the loop. Can you please recheck ?

Reply:
yes, missed this while merging the conflicts.